ext/drreg/drreg.c

```diff
@@ -249,6 +445,41 @@ get_spilled_value(void *drcontext, uint slot)
     }
 }
 
+#ifdef SIMD_SUPPORTED
+static bool
+get_indirectly_spilled_value(void *drcontext, reg_id_t reg, uint slot,
+                             OUT byte *value_buf, size_t buf_size)
+{
+    ASSERT(value_buf != NULL, "value buffer not initialised");
+    if (value_buf == NULL)
+        return false;
+    /* Get the size of the register so we can ensure that the buffer size is adequate. */
+    size_t reg_size = opnd_size_in_bytes(reg_get_size(reg));
+    ASSERT(buf_size >= reg_size, "value buffer too small in size");
+    if (buf_size < reg_size)
```

Comment:
It's an internal function and we already just asserted this assumption one line above. I think we don't need this check.

Reply:
What if we are running with a debug build?

Reply:
I see this as an internal error that we assert for and don't need a 'if' case for to handle it. I'd remove. If @derekbruening sees this differently and wants the 'if' then I apologize in advance that I made you remove it :)

Reply:
Done